### PR TITLE
Added vs code marketplace version badge

### PR DESCRIPTION
**Pull request by fsjoyti**

### CODE REVIEW

**Comment by fsjoyti** [comment]
Fixes issue #6

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Color Vision
 
+![](https://img.shields.io/visual-studio-marketplace/v/wilsonsio.color-vision?style=for-the-badge)
```
Comment (SgiobairOg) [inline README.md]:
```suggestion
![Visual Studio Marketplace Rating](https://img.shields.io/visual-studio-marketplace/v/wilsonsio.color-vision)
```

Comment (SgiobairOg) [reply README.md]: Two quick changes, one to add the alttext for the badge and one to remove the style for consistency. Thanks for the contribution.

**Review by SgiobairOg — CHANGES_REQUESTED** [review]

**Comment by fsjoyti** [comment]
Ok I revised my change. Let me know if that worked.

**Review by SgiobairOg — APPROVED** [review]
All good, thanks!

**Pull request merged**